static/css/components/tagging-menu.less

```diff
@@ -52,24 +52,25 @@
 // Menu option containers:
 .selection-container {
   height: 266px;
-  overflow-y: scroll;
+  overflow-y: auto;
 
   .subjects-search-results,
   .selected-tag-subjects {
     padding: 0 5px;
   }
-}
 
-::-webkit-scrollbar {
-  -webkit-appearance: none;
-  background-color: @lighter-grey;
-  width: 5px;
-}
+  scrollbar-width: thin;
```

Comment:
Can't figure out how to make this work in Chrome. Firefox handle scrollbar-thin correctly. Chrome makes scrollbar-thin override the next scrollbar style blocks. Oh well no harm no foul leaving it broken on firefox as it was for now.
```suggestion
```